Python_Engine/Query/Version.cs

```diff
@@ -31,6 +31,9 @@ namespace BH.Engine.Python
 {
     public static partial class Query
     {
+        [Description("Gets the Python version for a requested Python executable.")]
```

Comment:
```suggestion
        [Description("Gets the Python version for the requested Python executable.")]
```

Actually, because it's your first PR with the team, I have to find something... 😉